classes/class-wc-connect-help-provider.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Help_Provider' ) ) {
+
+	class WC_Connect_Help_Provider {
+
+		/**
+		 * @var WC_Connect_Service_Settings_Store
+		 */
+		protected $service_settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $service_settings_store ) {
+
+			$this->service_settings_store = $service_settings_store;
+
+			add_filter( 'woocommerce_admin_status_tabs', array( $this, 'status_tabs' ) );
+			add_action( 'woocommerce_admin_status_content_connect', array( $this, 'page' ) );
+
+		}
+
+		/**
+		 * Filters the WooCommerce System Status Tabs to add connect
+		 *
+		 * @param array $tabs
+		 * @return array
+		 */
+		public function status_tabs( $tabs ) {
+
+			if ( ! is_array( $tabs ) ) {
+				$tabs = array();
+			}
+			$tabs[ 'connect' ] = _x( 'Connect', 'The WooCommerce Connect brandname', 'woocommerce' );
+			return $tabs;
+
+		}
+
+		/**
+		 * Returns the form schema object for the entire help page
+		 *
+		 * @return object
+		 */
+		protected function get_form_schema() {
+
+			$form_schema = new stdClass();
+			$form_schema->properties = new stdClass();
+			return $form_schema;
+
+		}
+
+		/**
+		 * Returns the form layout array for the entire help page
+		 *
+		 * @return array
+		 */
+		protected function get_form_layout() {
+
+			return array(
+				(object) array(
+					'title' => __( 'Health', 'woocommerce' ),
```

Comment:
this might need `_x` with an explanation, because `Health` in this context might not be the same as `Health` in a medical context which i could see being potentially confusing to a translator